docs/docusaurus/docs/tutorials/quickstart/quickstart.md

```diff
@@ -67,8 +67,6 @@ Windows support for the open source Python version of GX is currently unavailabl
     ```
     The Expectation assumes the `pickup_datetime` column always contains data.  None of the column's values are null.
```

Comment:
We can probably remove this line.  I think it may be a hold over from when there was only one Expectation in the Quickstart, rather than two.  As it stands, it doesn't explicitly state that it is talking about the first Expectation of the two, and it seems superfluous to the explanation of that Expectation that was provided above the code block.

Reply:
@Rachel-Reverie the text has been removed.